### ISSUE TITLE
setting --app-index string to demo/index.html fails. 

### ISSUE THREAD

**Issue by fedler** [open-wc/open-wc#1543]
```command: es-dev-server -n -d -a demo/index.html```
```
response:
Serving files from '/Users/fedler/418-cafe/ux'.
Using history API fallback, redirecting route requests to '/demo/index.html'
Using auto compatibility mode, transforming code on older browsers based on user agent.
```
Server startup looks correct, but browsing to any files yeilds "Not Found" in browser.

**Comment by LarsDenBakker** [comment]
We need some more info to properly debug this. If there is a file at `demo/index.html` you should be able to see it by going to `demo/index.html` in the browser.

**Comment by fedler** [comment]
Yes I can browse to demo/index.html and that works. All other fake routes fail. Example: /fake or /demo/fake

**Comment by LarsDenBakker** [comment]
Indeed it only provides a fallback within the same directory scope.

**Comment by fedler** [comment]
we agree this is an issue right? The purpose of the fallback is to catch all unmatch resources for single page apps. Also, one of the more common patterns is to use /src /demo with built assets going to root

**Comment by LarsDenBakker** [comment]
I did this for a reason at the time, but I don't quite remember it now. Will need to think on it some more whether this is a good idea.
